Core.DomainModel/Organization/StsOrganizationChangelog.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Core.DomainModel.Organization
+{
+    public class StsOrganizationChangeLog: Entity
+    {
+        public virtual StsOrganizationConnection StsOrganizationConnection { get; set; }
```

Comment:
remember a default ctor setting the value of the collections (easier to code against than a possible.null)